alf/algorithms/actor_critic_algorithm.py

```diff
@@ -45,8 +45,7 @@ def __init__(self,
                  optimizer=None,
                  debug_summaries=False,
                  name="ActorCriticAlgorithm"):
-        """Create an ActorCriticAlgorithm.
-
+        """
```

Comment:
Because sphinx will combine the docstrings of the Class declaration and init function, we need to remove duplicate sentences accordingly.